community/index.md

```diff
@@ -33,26 +33,21 @@ Please report any issues on the given Test Kit's Issues page on [our GitHub](htt
 
 We welcome external contributions to our repositories. The information below explains expectations for contributors and the contribution process.
 
-#### Code of Conduct
-
+<h3 data-toc-skip class="h4">Code of Conduct</h3>
```

Comment:
```suggestion
### Code of Conduct
{:toc-skip}
```
Can use Kramdown attribute list definitions instead of commingling inline HTML. Add the definition to the top of the file: `{:toc-skip: .h4 data-toc-skip=""}` and add the `{:toc-skip}` after each markdown header